models/preles/R/runPRELES.jobsh.R

```diff
@@ -80,15 +80,15 @@ runPRELES.jobsh <- function(met.file, outdir, parameters, sitelat, sitelon, star
 
       ## Get PPFD from SW
       PPFD <- sw2ppfd(SW)  # PPFD in umol/m2/s
-      PPFD <- PPFD * 1e-06  # convert umol to mol
+      PPFD <- udunits2::ud.convert(PPFD, "umol m-2 s-1", "mol m-2 s-1")
 
       ## Format/convert inputs
       ppfd   <- tapply(PPFD, doy, mean, na.rm = TRUE)  # Find the mean for the day
       tair   <- udunits2::ud.convert(tapply(Tair, doy, mean, na.rm = TRUE), "kelvin", "celsius")  # Convert Kelvin to Celcius
       vpd    <- udunits2::ud.convert(tapply(VPD, doy, mean, na.rm = TRUE), "Pa", "kPa")  # pascal to kila pascal
       precip <- tapply(Precip, doy, sum, na.rm = TRUE)  # Sum to daily precipitation
       co2    <- tapply(CO2, doy, mean)  # need daily average, so sum up day
-      co2    <- co2 / 1e+06  # convert to ppm
+      co2    <- co2 / 1e+06  # convert to ppm. ANS: Convert from what? Mole-fraction to ppm is multiplying by 1e6, not dividing
```

Comment:
Is this an error that is currently causing invalid calculations? If so, who can resolve it?

Reply:
@tonygardella wrote this coupler, so I think he can check and resolve this.

Reply:
I'll fix it!